src/unix/linux_like/linux/musl/mod.rs

```diff
@@ -164,6 +185,26 @@ s! {
         __f_spare: [::c_int; 6],
     }
 
+    pub struct statvfs64 {
+        pub f_bsize: ::c_ulong,
+        pub f_frsize: ::c_ulong,
+        pub f_blocks: ::fsblkcnt64_t,
+        pub f_bfree: ::fsblkcnt64_t,
+        pub f_bavail: ::fsblkcnt64_t,
+        pub f_files: ::fsfilcnt64_t,
+        pub f_ffree: ::fsfilcnt64_t,
+        pub f_favail: ::fsfilcnt64_t,
+        #[cfg(target_endian = "little")]
+        pub f_fsid: ::c_ulong,
+        #[cfg(target_pointer_width = "32")]
+        __f_unused: ::c_int,
```

Comment:
I don't see this field in the definition at https://github.com/kraj/musl/blob/ffb23aef7b5339b8c3234f4c6a93c488dc873919/include/sys/statvfs.h#L14-L28, any idea where it comes from?

Reply:
Good question, I think I pulled it from the `b32/arm/mod.rs` definition.  Looking at pahole on a 32-bit musl libc I can see:

```
struct statvfs {
        long unsigned int          f_bsize;              /*     0     4 */
        long unsigned int          f_frsize;             /*     4     4 */
        fsblkcnt_t                 f_blocks;             /*     8     8 */
        fsblkcnt_t                 f_bfree;              /*    16     8 */
        fsblkcnt_t                 f_bavail;             /*    24     8 */
        fsfilcnt_t                 f_files;              /*    32     8 */
        fsfilcnt_t                 f_ffree;              /*    40     8 */
        fsfilcnt_t                 f_favail;             /*    48     8 */
        long unsigned int          f_fsid;               /*    56     4 */

        /* XXX 4 bytes hole, try to pack */

        /* --- cacheline 1 boundary (64 bytes) --- */
        long unsigned int          f_flag;               /*    64     4 */
        long unsigned int          f_namemax;            /*    68     4 */
        int                        __reserved[6];        /*    72    24 */

        /* size: 96, cachelines: 2, members: 12 */
        /* sum members: 92, holes: 1, sum holes: 4 */
        /* last cacheline: 32 bytes */
};
```

So this `__f_unused` is simply explicit padding on 32-bit systems.  We don't seem to add explicit padding elsewhere so I'll remove this.

Reply:
Ah, it _is_ explicit padding... https://git.musl-libc.org/cgit/musl/tree/include/sys/statvfs.h?h=v1.1.24 has:

```
#if __BYTE_ORDER == __LITTLE_ENDIAN
	unsigned long f_fsid;
	unsigned :8*(2*sizeof(int)-sizeof(long));
#else
	unsigned :8*(2*sizeof(int)-sizeof(long));
	unsigned long f_fsid;
#endif
```

Where those non-standard lines are "anonymous bitfields" that adds padding so the `f_fsid` field is as big as two `int`s.